src/index.tsx

```diff
@@ -1983,10 +1983,17 @@ export class App extends React.Component<any, AppState> {
     e.preventDefault();
     const { deltaX, deltaY } = e;
 
-    this.setState({
-      scrollX: this.state.scrollX - deltaX / this.state.zoom,
-      scrollY: this.state.scrollY - deltaY / this.state.zoom,
-    });
+    if (e[KEYS.META]) {
+      this.setState(({ zoom }) => ({
+        zoom: Math.max(Math.min(zoom - deltaY / 100, 2), 0.1),
```

Comment:
There is a clamp zoom function [here](https://github.com/excalidraw/excalidraw/blob/master/src/actions/actionCanvas.tsx#L59) that we should be using. Probably we want to move it to `src/scene/zoom.ts`.

Reply:
Updated, thanks!